pxtsim/runtime.ts

```diff
@@ -1097,6 +1102,26 @@ namespace pxsim {
             c.numstops++;
         }
 
+        startIdle() {
+            // schedules handlers to run every 20ms
+            if (this.idleTimer === undefined) {
+                this.idleTimer = setInterval(() => {
+                    if (!this.running || this.pausedOnBreakpoint) return;
+                    const bus = this.board.bus;
+                    if (bus)
+                        bus.queueIdle();
+                });
```

Comment:
why not specify timeout value? the minimum is 10ms.

Reply:
oopsy